build/buildrelease.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$1" = "" ]]
```

Comment:
Prefer `if [[ $# -ne 1 ]]`.

(If you're using bash, `[[` has less sharp edges that `test` / `[`, and you may as well check the number of arguments directly.)

Also prefer to format these as:

```bash
if expr; then
  ...
fi
```

rather than place the `then` on the next  line.